external-crates/move/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -215,10 +214,9 @@ impl Interpreter {
                 }
                 ExitCode::Call(fh_idx) => {
                     let func = resolver.function_from_handle(fh_idx);
-                    // Compiled out in release mode
                     #[cfg(debug_assertions)]
```

Comment:
This needs to be `gas-profiler` feature-flag enabled. Otherwise you'll get compilation errors in release mode.